03-serverspec/README.md

```diff
@@ -1,3 +1,14 @@
+# Installation
+
+Just run
+```bash
+mvn install
+```
+in this directory to install the parent `pom.xml` used for docker builds.
```

Comment:
"in this directory" is not correct. The parent pom resides in 00-. What was your orignal intent? May I drop this safely?

Reply:
I needed to call "mvn install" for the following commands to work. The rest is a copy'n'paste-error. :)

Reply:
ah ... now I get it. Completely escaped me that I never lost a word there ... I"ll need to fix two smallish issues after merging, but thank you very much for spotting that!